### PR TITLE
Style

### PR DESCRIPTION
**Pull request by tay-aitken**
## Overview

- Updating the style tab with class names
- making sure the style tab is consistent across all components 
- address any issues from [563](https://github.ibm.com/carbon/issues/issues/563) (internal issue)

### CODE REVIEW

**Comment by tay-aitken** [comment]
There are a few pages with particularly long classes that we may need to truncate. I think we can wait on doing that until we convert the site over to Gatsby.

**Comment by aagonzales** [comment]
Is `text-01`= 152934 or 152935?? 

In the theme and on the website color page its #15293**4**
https://github.com/carbon-design-system/carbon-themes/blob/master/themes/carbon.scss  

But in the kit and on a lot of the style tabs its #15293**5**

**Comment by tay-aitken** [comment]
Do you remember what it was when it was initially established? Is there a place we could check on that?

File: src/content/components/button/style.md [hunk 1]
```diff
@@ -103,3 +102,12 @@ A Button cannot have any element or text within 16 pixels / 1 rem of its borders
   </div>
 </div>
 _Spacing measurements for various Button types | px / rem_
+
+### Recommended
+
+The following specs are not built into any of the Button components but are recommended by design as the proper amount of space to distance Buttons should be from one another.
```
Comment (aagonzales) [inline src/content/components/button/style.md]: I think there's something off at the end of this sentence.

File: src/content/components/select/style.md [hunk 2]
```diff
@@ -2,74 +2,99 @@
...
+|`.bx--select-input:focus`| box-shadow    | 2px       | - |
 
 ![Structure and spacing measurements for Select](images/select-style-2.png)
 _Structure and spacing measurements for Select | px / rem_
 
-### States
+### Recommended
 
-**Open:** Select menu styles are determined by browser. Each browser has a unique style. Shown in the example images is the Chrome select style.
+The following specs are not built into the Select component but are recommended by design as the proper amount of space to distance Select from another form element or component.
```
Comment (aagonzales) [inline src/content/components/select/style.md]: "...the proper amount of space to distance Select from another form element or component." Something off with this sentence as well or am I misunderstanding? Its the "space to distance"  part that's throwing me off.

**Comment by aagonzales** [comment]
@tay-aitken text-01 was the Blue 90 from the original IDL color palette at that defined it as `#152935`

File: src/content/components/breadcrumb/style.md [hunk 3]
```diff
@@ -24,10 +22,35 @@ _Breadcrumb typography treatment example_
...
+| Class                        | Property   | px/rem| Spacing token|
+|------------------------------|------------|-------|--------------|
+| `.bx--breadcrumb-item::after`| margin-left| 16 / 1| $spacing-md  |
+
 
 ![Truncated breadcrumb dropdown example](images/breadcrumb-style-2.png)
 _Structure and spacing measurements for Breadcrumb | px / rem_
+
+## Recommended
+Truncated breadcrumbs are not currently built into the Breadcrumb component.The following colors from the [Overflow Menu](/overflow/style) are recommended for the overflow Breadcrumb list.
```
[the remark this answers is not in the document]
Comment (tay-aitken) [reply src/content/components/breadcrumb/style.md]: Fixed!

File: src/content/components/button/style.md (hunk 1 above)
Comment (tay-aitken) [reply src/content/components/button/style.md]: Fixed!

File: package-lock.json [hunk 4]
```diff
@@ -11198,9 +11198,15 @@
       "dev": true
     },
     "p-limit": {
+<<<<<<< HEAD
```
Comment (asudoh) [inline package-lock.json]: Seems that merge conflict is left here?

File: src/app/components/internal/SideNav/SideNav.js [hunk 5]
```diff
@@ -259,119 +259,145 @@ class SideNav extends Component {
...
+      );
+    }
   }
-}
 
-export default SideNav;
+  export default SideNav;
+
+
+            // <Button
```
Comment (asudoh) [inline src/app/components/internal/SideNav/SideNav.js]: Did you intend to remove these lines...?

Comment (tay-aitken) [reply src/app/components/internal/SideNav/SideNav.js]: No I didn't. Let me see if I can add it back in.

File: package-lock.json (hunk 4 above)
Comment (tay-aitken) [reply package-lock.json]: Addressed in latest commit

**Comment by tay-aitken** [comment]
🎉

**Pull request merged**